packages/flutter/lib/src/painting/borders.dart

```diff
@@ -21,6 +21,27 @@ enum BorderStyle {
   // if you add more, think about how they will lerp
 }
 
+/// The direction of the stroke to be painted for a [BorderSide] in a [Border].
```

Comment:
```suggestion
/// The relative position of the stroke on a [BorderSide] in a [Border].
```

Reply:
question: do I leave it that way or write `in a Border or OutlinedBorder`? Technically it belongs to the two.

Reply:
Sure, add in the `OutlinedBorder`.